mangaki/mangaki/models.py

```diff
@@ -240,20 +240,43 @@ class WorkTitle(models.Model):
                             blank=True,
                             db_index=True)
 
+    class Meta:
+        unique_together = ('title', 'language')
+
     def __str__(self):
         return "%s" % self.title
 
+UNK_LANG_VALUE = 'x-unk'
```

Comment:
I think this should just be `None`, to be honest (since version 1.11, Django properly handles `CharField` with `is_null=True`).